src/lib/structures/Piece.ts

```diff
@@ -79,11 +72,11 @@ export class Piece {
 	}
 
 	/**
-	 * The context given by the store.
-	 * @see Store.injectedContext
+	 * A reference to the [[container]] object for ease of use.
```

Comment:
```suggestion
	 * A reference to the {@link container} object for ease of use.
```

For TS 4.3

Also after changing it run `yarn docs` to validate that typedoc doesn't shit itself. It shouldn't, but it's worth a check anyway.